Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Startup.cs

```diff
@@ -88,9 +89,8 @@ public void Configure(IApplicationBuilder app)
                 */
             });
 
-#if DEBUG
-            TelemetryConfiguration.Active.DisableTelemetry = true;
-#endif
+            if (Debugger.IsAttached)
```

Comment:
See above

Reply:
In this case we want to avoid telemetry submission on local run  - so it's ok.